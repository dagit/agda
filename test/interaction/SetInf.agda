{-# OPTIONS --universe-polymorphism #-}

module SetInf where

id : ∀ {A} → A → A
id x = x
