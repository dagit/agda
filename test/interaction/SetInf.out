agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*All Goals*" "_7 : SetÏ‰  [ at SetInf.agda:5,14-19 ]
" nil)
agda2_mode_code (agda2-goals-action '())
